src/components/Navbar.js

```diff
@@ -0,0 +1,53 @@
+//@flow strict
+
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+
+import type {LinkType} from './LinkType.js';
+
+type Props = {
+	styleName: string,

```

Comment:
This shouldn't be string. It should be 'nobtn'|'navbar' or whatever the legal styles are. Someone like me who doesn't know what this is used for is likely to pass in the wrong string if we want to use it.